### DISCUSSION TITLE
Version Number?

### DISCUSSION THREAD

**Discussion by NeuroDawg** [dani-garcia/vaultwarden#3641] [Q&A]
Can someone explain the version numbers to me? When I look here, at these github pages, it says the latest version is 1.28.1, released on April 2, 2023.

I've installed the "vaultwarden/server:latest" image in Docker, and on the login page for my server, it says it version 2023.3.0.

So, why the difference? What am I missing in trying to understand this numbering?

**Comment by BlackDex** [comment]
The version number on the web-vault is that of the web-vault, not Vaultwarden.

The Vaultwarden version can be checked via the admin interface.
Or via `/api/config`
